code/lib/csf-tools/src/ConfigFile.ts

```diff
@@ -506,6 +506,34 @@ export class ConfigFile {
     }
   }
 
+  /**
+   * Specialized helper to remove addons or other array entries
+   * that can either be strings or objects with a name property.
+   */
+  removeEntryFromArray(path: string[], value: string) {
```

Comment:
This function doesn't take care of the pnp workaround to resolve absolute paths for packages.

E.g.:

```
import type { StorybookConfig } from '@storybook/sveltekit';

import { join, dirname } from 'path';

/**
 * This function is used to resolve the absolute path of a package.
 * It is needed in projects that use Yarn PnP or are set up within a monorepo.
 */
function getAbsolutePath(value: string): any {
  return dirname(require.resolve(join(value, 'package.json')));
}
const config: StorybookConfig = {
  stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|mjs|ts|tsx)'],
  addons: [
    getAbsolutePath('@storybook/addon-links'),
    getAbsolutePath('@storybook/addon-essentials'),
    getAbsolutePath('@storybook/addon-interactions')
  ],
  framework: {
    name: getAbsolutePath('@storybook/sveltekit'),
    options: {}
  }
};
```

Reply:
Great catch! Will update